charts/intel-gpu-resource-driver/values.yaml

```diff
@@ -1,6 +1,6 @@
 # Default values for intel-gpu-resource-driver.
 nameOverride: ""
-namespaceOverride: "intel-gpu-resource-driver"
+namespaceOverride: ""
```

Comment:
Why you're removing the current default?

Reply:
At least you need to add the necessary Helm options to README, so that users do not install it to `default` ns by default, see: https://github.com/intel/intel-resource-drivers-for-kubernetes/blob/main/doc/gpu/USAGE.md#helm-charts